### DISCUSSION TITLE
The dependency was already listed in devDependencies. If you want to make it a prod dependency, then move it manually.

### DISCUSSION THREAD

**Discussion by cowwoc** [pnpm/pnpm#7373] [Q&A]
When I run `pnpm install` I keep on getting this annoying warning message at the end which I'd love to understand (so I can resolve it). Here is the output I see:

```
Lockfile is up to date, resolution step is skipped
Packages: +520
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Progress: resolved 520, reused 520, downloaded 0, added 520, done

dependencies:
+ @cowwoc/requirements 3.3.2
+ lit 3.1.0
+ lodash 4.17.21

devDependencies:
+ @eslint/js 8.55.0
+ @types/chai 4.3.11
+ @types/eslint 8.44.8
+ @types/glob 8.1.0
+ @types/google.accounts 0.0.14
+ @types/lodash 4.14.202
+ @types/minimist 1.2.5
+ @types/mocha 10.0.6
+ @types/node 20.10.2
+ @typescript-eslint/eslint-plugin 6.13.1
+ @typescript-eslint/parser 6.13.1
+ autoprefixer 10.4.16
+ browser-sync 2.29.3
+ browserslist 4.22.2
+ c8 8.0.1
+ chai 4.3.10
+ cheerio 1.0.0-rc.12
+ chokidar 3.5.3
+ esbuild 0.19.8
+ esbuild-plugin-browserslist 0.10.0
+ eslint 8.55.0
+ eslint-import-resolver-typescript 3.6.1
+ eslint-plugin-import 2.29.0
+ eslint-plugin-tsdoc 0.2.17
+ fast-glob 3.3.2
+ install 0.13.0
+ minimist 1.2.8
+ mocha 10.2.0
+ pnpm 8.11.0
+ postcss 8.4.32
+ postcss-nested 6.0.1
+ tailwindcss 3.3.5
+ ts-node 10.9.1
+ tslib 2.6.2
+ tsx 4.6.2
+ typedoc 0.25.4
+ typedoc-plugin-missing-exports 2.1.0
+ typescript 5.3.2
+ typescript-eslint 0.0.1-alpha.0
+ winston 3.11.0

The dependency was already listed in devDependencies.
If you want to make it a prod dependency, then move it manually.
```

Does anyone know what `The dependency was already listed in devDependencies. If you want to make it a prod dependency, then move it manually.` refers to and how I can fix it? I'm not asking `pnpm` to move a dependency into `devDependencies` so I don't know why it keeps on claiming I am :)

**Comment by its-monotype** [comment]
It drives me crazy. I've checked to make sure that I haven't accidentally duplicated any dependencies, but everything seems to be correct. I even upgraded node to version 20, but the issue persists. I also tried installing with npm and didn't encounter any errors. Maybe it's after some pnpm update...

**Reply by its-monotype** [reply]
I discovered that the issue only occurs in version 8.11.0. I tested version 8.10.5 and did not encounter this warning.

**Reply by its-monotype** [reply]
check it out https://github.com/pnpm/pnpm/compare/v8.10.5...v8.11.0#diff-439970ce8a3bde9ed7fa780c9bfa222d021210b9a608b0f72ab03a936902d812

**Reply by its-monotype** [reply]
Check it out https://github.com/pnpm/pnpm/commit/45bdc79b17156a2464f71c2de4c0ae0dc31b2e7f

**Comment by its-monotype** [comment]
There is existing PR with fix: https://github.com/pnpm/pnpm/pull/7348

**Comment by cowwoc** [comment]
Thanks for tip! I'll follow that PR.
